src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -38,11 +38,12 @@ RaftReplDev::RaftReplDev(RaftReplService& svc, superblk< raft_repl_dev_superblk
         }
 
         if (m_rd_sb->is_timeline_consistent) {
-            logstore_service().open_log_store(LogStoreService::CTRL_LOG_FAMILY_IDX, m_rd_sb->free_blks_journal_id,
-                                              false, [this](shared< HomeLogStore > log_store) {
-                                                  m_free_blks_journal = std::move(log_store);
-                                                  m_rd_sb->free_blks_journal_id = m_free_blks_journal->get_store_id();
-                                              });
+            logstore_service()
+                .open_log_store(LogStoreService::CTRL_LOG_FAMILY_IDX, m_rd_sb->free_blks_journal_id, false)
+                .thenValue([this](auto log_store) {
+                    m_free_blks_journal = std::move(log_store);
```

Comment:
is `m_free_blks_journal` for future usage? Any free_blks (calling from HomeObject) will be journalized by this log store and truncated after cp was done upto certain point? I didn't see where we used it yet in homestore code yet, probably this part is going to be implemented?

Reply:
m_free_blks_journal is needed if we set that we need `timeline_consistent`, which is not required for HomeObject, since there is no mutation for the key. It is needed for HomeBlocks.